src/screens/WalletMigrationArchanova/Review/WalletMigrationArchanovaReview.js

```diff
@@ -99,6 +117,8 @@ function WalletMigrationArchanovaReview() {
   };
 
   const hasEnoughGas = hasEnoughEthBalance(walletBalances, fee);
+  const feeInWei = ethToWei(fee).toString();
+  const highFee = isHighGasFee(chain, feeInWei, null, chainRates, currency, gasThresholds);
```

Comment:
I think i've seen this before... could this be a utility function?

Reply:
Hmm I couldn't find any functions that flagged high gas prices, which is why I made one, but I'll have another look around.

The `ethToWei` function is something I found only being used in a `storybook.js` file, but I did some basic testing and it seems to work. I couldn't find any utils files that had a similar function though.